modules/node_modules/@ncigdc/utils/survivalplot.js

```diff
@@ -20,8 +20,8 @@ export const enoughData = (data: Object) => (
   data.results.length && data.results.every(r => r.donors.length >= MINIMUM_CASES)
 );
 
-async function fetchCurves(filters: ?Array<Object>): Promise<Object> {
```

Comment:
Useful as a poc/mvp-demo, but I imagine a researcher would expect the graph to either show all results with 0 truncation, or use some kind of sampling that would result in a graph that is representative of the population

It's a problem we should definitely solve before putting this into production